content/en/docs/Features/ambient.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Istio Ambient Mesh
+description: Visualizing Ambient Mesh with Kiali
+---
+
+Kiali provides visualization for Ambient Mesh components: 
+
+* [Control Plane Ambient Mesh](#control-plane-ambient-mesh)
+* [Ambient nampespace](#ambient-namespace)
+* [Workloads in Mesh](#workloads-in-ambient-mesh)
+* [Ambient Telemetry](#ambient-telemetry)
+
+{{% alert color="warning" %}}
+The Kiali Ambient features, as well as Ambient Mesh, are evolving. Some of these features are in alpha status. For enhancements or detected issues, don’t hesitate to open a [GitHub issue](https://github.com/kiali/kiali/issues/new/choose). 
+{{% /alert %}}
+
+### Control Plane Ambient Mesh
+
+When the control plane is in Ambient mode, Kiali will show an Ambient label on the control plane card. 
+This label indicates that Kiali has detected a ztunnel (the L4 component for Ambient) in the control plane.
+
+![Ambient Control Plane](/images/documentation/features/ambient/ambient-control-plane.png)
+
+### Ambient Namespace
+
+When a namespace is included in Ambient Mesh because it has the Istio Ambient label, Kiali will show the Ambient label on that namespace card: 
+
+![Ambient Data Plane](/images/documentation/features/ambient/ambient-data-plane.png)
+
+### Workloads in Ambient Mesh
+
+When a workload, application, or service is part of the Ambient Mesh, a label will appear in the namespace details. When hovering over this label, further information about the workload will be displayed:
+
+* In Mesh: Indicating that it was included in Ambient, and the traffic is redirected to ztunnel to provide L4 features (L4 authorization and telemetry, and encrypted data transport)
+
+  ![Workload Captured by Ambient](/images/documentation/features/ambient/ztunnel-captured-pod.png)
+
+* In Mesh with waypoint enabled: Additionally, it can include the L7 label which means that a waypoint proxy is deployed (providing additional L7 capabilities):
```

Comment:
```suggestion
* In Mesh with waypoint enabled: Additionally, it can include the L7 badge which means that a waypoint proxy is deployed (providing additional L7 capabilities):
```